tutorials/02-Basic Operations.ipynb

```diff
@@ -0,0 +1,1822 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "816450d3-82ba-4522-ade3-f106aad29a6b",
+   "metadata": {},
+   "source": [
+    "We also have experimental $LaTeX$ parser\n",
+    "\n",
+    "https://docs.sympy.org/latest/modules/parsing.html#experimental-mathrm-latex-parsing\n",
+    "\n",
+    "You would need to install `antlr` which is difficult to cover in this tutorial,\n",
+    "however, this can be useful for some interested readers."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "483ca70d-5c7e-4ea2-8bf9-90f579adeecd",
+   "metadata": {},
+   "source": [
+    "# Printing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f9409e77-b5e8-412e-899b-176a8512f302",
+   "metadata": {},
+   "source": [
+    "## Math printers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d13fd39-d188-4dc2-8249-fa6cd9a96a61",
+   "metadata": {},
+   "source": [
+    "Although you mostly see the LaTeX typesetted expression here,\n",
+    "\n",
+    "It is SymPy's `latex` function that does real work behind\n",
+    "\n",
+    "(The magic is done by `_repr_latex_` interface from IPython)\n",
+    "\n",
+    "Calling `latex` is useful if you want to copy the typesetted SymPy expressions into your LaTeX article."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "id": "cbc0016e-d224-4403-a179-244db8fc675a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'e^{\\\\sin{\\\\left(x^{2} \\\\right)}}'"
+      ]
+     },
+     "execution_count": 44,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "latex(exp(sin(x**2)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c6ceb1a-3e2d-4055-b8f1-4df0f1893d01",
+   "metadata": {},
+   "source": [
+    "(Be careful that Python escapes the backslashes if string is represented)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "id": "32d7f5cd-0043-4852-99ee-bf4257be5a76",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "e^{\\sin{\\left(x^{2} \\right)}}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(latex(exp(sin(x**2))))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87b4180a-b452-44ad-9f5c-690bd8c3ee12",
+   "metadata": {},
+   "source": [
+    "And you can use `pprint` (pretty printing) if you still need to typeset the mathematical expressions\n",
+    "with the ascii art (like the graphing calculator).\n",
+    "\n",
+    "(However, you would only need `latex` for most of the cases)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 54,
+   "id": "c0be1ee5-3f8a-4e51-8d0b-8582714ab488",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    / 2\\\n",
+      " sin\\x /\n",
+      "e       \n"
+     ]
+    }
+   ],
+   "source": [
+    "pprint(exp(sin(x**2)), use_unicode=False)"
```

Comment:
Why do you use `use_unicode=False`?

Reply:
The unicode doesn't get printed properly unless dejavu sans mono is installed, which I want to avoid the complication.